diff --git a/lib/mkmf.rb b/lib/mkmf.rb
index eabccd4..a4dc9c5 100644
--- a/lib/mkmf.rb
+++ b/lib/mkmf.rb
@@ -2316,7 +2316,7 @@ def create_makefile(target, srcprefix = nil)
 ORIG_SRCS = #{orig_srcs.collect(&File.method(:basename)).join(' ')}
 SRCS = $(ORIG_SRCS) #{(srcs - orig_srcs).collect(&File.method(:basename)).join(' ')}
 OBJS = #{$objs.join(" ")}
-HDRS = #{hdrs.map{|h| '$(srcdir)/' + File.basename(h)}.join(' ')}
+HDRS = #{hdrs.sort.map{|h| '$(srcdir)/' + File.basename(h)}.join(' ')}
 LOCAL_HDRS = #{$headers.join(' ')}
 TARGET = #{target}
 TARGET_NAME = #{target && target[/\A\w+/]}
diff --git a/lib/rubygems/ext/ext_conf_builder.rb b/lib/rubygems/ext/ext_conf_builder.rb
index 88be7ec..8191b0a 100644
--- a/lib/rubygems/ext/ext_conf_builder.rb
+++ b/lib/rubygems/ext/ext_conf_builder.rb
@@ -14,7 +14,8 @@ class Gem::Ext::ExtConfBuilder < Gem::Ext::Builder
   FileEntry = FileUtils::Entry_ # :nodoc:

   def self.build(extension, dest_path, results, args=[], lib_dir=nil)
-    tmp_dest = Dir.mktmpdir(".gem.", ".")
+    tmp_dest = ".gem."
+    Dir.mkdir(tmp_dest)

     # Some versions of `mktmpdir` return absolute paths, which will break make
     # if the paths contain spaces. However, on Ruby 1.9.x on Windows, relative
